src/jdk.jlink/share/classes/jdk/tools/jlink/resources/plugins.properties

```
@@ -59,38 +59,30 @@ Class optimization: convert Class.forName calls to constant loads.
 class-for-name.usage=\
 \  --class-for-name          Class optimization: convert Class.forName calls to constant loads.
 
-compress.argument=<0|1|2>[:filter=<pattern-list>]
+compress.argument=<0|1|2|zip-[0-9]>[:filter=<pattern-list>]
 
 compress.description=\
-Compress all resources in the output image.\n\
-Level 0: No compression\n\
-Level 1: Constant string sharing\n\
-Level 2: ZIP.\n\
-An optional <pattern-list> filter can be specified to list the pattern of\n\
-files to be included.
+Compression to use in compressing resources:\n\
+Accepted values are:\n\
+zip-[0-9], where zip-0 provides no\n\
+compression, and zip-9 provides the\n\
+best compression. Default is zip-6.\n\
+Deprecated values to be removed in a future release:\n\
```

```suggestion
Compression to use in compressing resources.
```

`compress.description` is just the description of the plugin.   `compress.usage` shows the accepted values.